protocol/packages/swap/src/astroport.rs

```diff
@@ -23,37 +30,43 @@ use sdk::{
     },
     cosmwasm_std::{self, Coin as CwCoin, Decimal},
 };
-use serde::{Deserialize, Serialize};
 
+// TODO change visibility to private
 pub type RequestMsg = MsgExecuteContract;
```

Comment:
I believe this type is not used outside of this module so please resolve the TODO

Reply:
That's true. Thanks!